L6_MaxProductOfThree.py

```diff
@@ -78,11 +78,15 @@ def solution(A):
     :param A: list[int]
     :return: [int]
     """
-    A.sort()
-    return max(
-        A[0] * A[1] * A[-1],
-        A[-3] * A[-2] * A[-1]
-    )
+    max=0
+    
+    a=sorted(a)
```

Comment:
The input parameter is called "A", upper case, not "a", so this will not compile.